libmscore/layout.cpp

```diff
@@ -3720,13 +3720,15 @@ System* Score::collectSystem(LayoutContext& lc)
             }
 
       hideEmptyStaves(system, lc.firstSystem);
+#if 0
```

Comment:
Looks like dead code to me?

Reply:
Not anymore, this code is restore in latest <code>3.x</code> commits.

Reply:
Nope, I'm on the latest

Reply:
8c7af0b2b removes an `if (!allShown) `

Reply:
So exactly the last commit before my PR